packages/client/src/components/ToolsGrid/ToolsGrid.component.js

```diff
@@ -63,6 +63,7 @@ export const ToolsGrid = () => {
             groupSizeMin={tool.group_size_min}
             groupSizeMax={tool.group_size_max}
             pitch={tool.pitch}
+            categories={tool.categories}
```

Comment:
Notice how each prop here just does `tool.PROPERTY`, this means that the ToolItem really cares about tools. So each time you add a new property to tool, you need to update this code but ToolsGrid doesn't really care about the details of tools.

So instead, change the interface to;
`<ToolItem tool={tool} />`